File: Clubz/Models/Member.cs
namespace Clubz.Models;

using System.ComponentModel.DataAnnotations;

public class Member
{
  public int Id { get; set; }
  public int ClubId { get; set; }
  [Required]
  public string Forename { get; set; }
  [Required]
  public string Surname { get; set; }
  [Required]
  public string EmailAddress { get; set; }

}

File: web/src/lib/schema/member.ts
import { z } from 'zod'

const Member = z.object({
	Id: z.number(),
	ClubId: z.number(),
	Forename: z.string().min(1).max(50),
	Surname: z.string().min(1).max(50),
	EmailAddress: z.string().min(1).max(50)
})

export { Member }
